app/hooks/use-wallet-balance.ts

```diff
@@ -33,8 +33,12 @@ export const useWalletBalance = (): {
   const wallet = data?.me?.defaultAccount?.wallets?.[0]
 
   if (!wallet) {
-    // If this is thrown, it means Apollo Cache is not able to read the MAIN_QUERY from the cache (because of missing fields)
-    throw new Error("Something went wrong")
+    // If we get here, it means Apollo Cache is not able to read the MAIN_QUERY from the cache (because of missing fields)
+    return {
+      satBalance: 0,
```

Comment:
could we use NaN instead of 0?

it may be better to show that something is wrong, instead of showing the balance of the user to 0 (which could make a user think his balance is gone)